docs/source/extension/extension_points.rst

```diff
@@ -621,23 +621,12 @@ Here is an example of using the ``closeAndCleaners`` semantic menu item:
 .. code:: typescript
 
     mainMenu.fileMenu.closeAndCleaners.add({
-      tracker,
-      action: 'Shutdown',
-      name: 'My Activity',
-      closeAndCleanup: current => {
-        current.close();
-        return current.shutdown();
-      }
+      id: 'notebook:close-and-shutdown',
+      isEnabled: (w: Widget) => tracker.currentWidget !== null && tracker.has(w)
     });
 
-In this example, ``tracker`` is a :ref:`widget-tracker`, which allows the menu
-item to determine whether to delegate the menu command to your activity,
-``name`` is a name given to your activity in the menu label,
-``action`` is a verb given to the cleanup operation in the menu label,
-and ``closeAndCleanup`` is the actual function that performs the cleanup operation.
-So if the current application activity is held in the ``tracker``,
-then the menu item will show ``Shutdown My Activity``, and delegate to the
-``closeAndCleanup`` function that was provided.
+In this example, ``tracker`` is a :ref:`notebook-tracker`, which allows the menu
```

Comment:
Did you forget to change the reference id or was this change unintentional?

```suggestion
In this example, ``tracker`` is a :ref:`widget-tracker`, which allows the menu
```

Reply:
It was changed because with this example in the case of the `notebook:close-and-shutdown` it will correspond to a notebook tracker:

https://github.com/jupyterlab/jupyterlab/blob/1311f0452de2dfa654fc09ea285e9f1a95c8fdef/packages/notebook-extension/src/index.ts#L1865

https://github.com/jupyterlab/jupyterlab/blob/1311f0452de2dfa654fc09ea285e9f1a95c8fdef/packages/notebook-extension/src/index.ts#L1573

But yeah we can keep the more generic `widget-tracker` in the docs.

Reply:
We can definitely change it. It is just not found in the doc; hence the CI error. For reference the `widget-tracker` anchor is defined there in the doc: https://github.com/jupyterlab/jupyterlab/blob/main/docs/source/extension/extension_points.rst#L880

Reply:
Switched back to `widget-tracker` for consistency :+1: